modules/utxo-lib/src/bitgo/UtxoTransaction.ts

```diff
@@ -8,29 +8,36 @@ export function varSliceSize(slice: Buffer): number {
   return varuint.encodingLength(length) + length;
 }
 
-export class UtxoTransaction extends bitcoinjs.Transaction {
+export class UtxoTransaction<TNumber extends number | bigint = number> extends bitcoinjs.Transaction<TNumber> {
   static SIGHASH_FORKID = 0x40;
   /** @deprecated use SIGHASH_FORKID */
   static SIGHASH_BITCOINCASHBIP143 = UtxoTransaction.SIGHASH_FORKID;
 
-  constructor(public network: Network, transaction: bitcoinjs.Transaction = new bitcoinjs.Transaction()) {
+  constructor(
+    public network: Network,
+    transaction: bitcoinjs.Transaction<TNumber> = new bitcoinjs.Transaction<TNumber>()
+  ) {
     super();
     this.version = transaction.version;
     this.locktime = transaction.locktime;
     this.ins = transaction.ins.map((v) => ({ ...v }));
     this.outs = transaction.outs.map((v) => ({ ...v }));
   }
 
-  static fromBuffer(
+  static fromBuffer<TNumber extends number | bigint = number>(
     buf: Buffer,
     noStrict: boolean,
+    amountType: 'number' | 'bigint' = 'number',
```

Comment:
I noticed here that I changed the order of the arguments though, going to reorder so that `amountType` is at the end

Reply:
Oh, this isn't actually possible because `bitcoinjs-lib` has the third argument as `amountType`

Reply:
I think this is an acceptable breaking change, going to record in commit message